appveyor.yml

```diff
@@ -8,8 +8,9 @@ environment:
   - nodejs_version: "8"
   - nodejs_version: "10"
   - nodejs_version: "11"
```

Comment:
I think `11` can be removed (EOL - july 2019), aren't?

Reply:
All v0.22.x releases will (continue to) support Node 11.